### PR TITLE
Configuration as Code YAML Schema

### PR DESCRIPTION
**Pull request by bitwiseman**
### NOT READY FOR DRAFT

@ndeloof @ewelinawilkosz 
I pulled the text from JEP-201 to here.  This is a placeholder for the Configuration as Code YAML Schema JEP.  

Before this can be approved as Draft,  it seems to me that most of the Specification needs rewriting/filling in - describing how the schema we be generated and the APIs for customizing the schema when the automatically generated schema is not sufficient. 

However, I don't have the expertise to write that. 

You can either update this PR or pull the branch and start your own.

### CODE REVIEW

**Comment by bitwiseman** [comment]
@ndeloof @ewelinawilkosz This needs to be updated.  Please make the edits and submit a PR.

**Comment by ndeloof** [comment]
@bitwiseman yes I know, I will look into this JEP effort next week

**Comment by ewelinawilkosz** [comment]
I will have a look today and update where it makes sense, thanks @bitwiseman!

File: jep/201/README.adoc [hunk 1]
```diff
@@ -1,296 +0,0 @@
-= JEP-201: Jenkins Configuration as Code
```
Comment (oleg-nenashev) [inline jep/201/README.adoc]: hmm, why does it delete the JEP entirely?

Comment (ndeloof) [reply jep/201/README.adoc]: I guess @bitwiseman did moved jep-201 REAME file into this new template file, not just the content, so from a git history pov the initial JEP has been removed.

Comment (ndeloof) [reply jep/201/README.adoc]: force-pushed to fix initial commit

**Comment by bitwiseman** [comment]
Approving as JEP-209.  I think this needs more work and discussion, but it is ready for draft.

**Pull request merged**